dotnet/api/Presentation/Web/Validators/Jobs/JobValidator.cs

```diff
@@ -34,5 +35,10 @@ private bool BeValidJsonStringThatConvertsToListOfObjects<TDto>(TDto dto, string
                 return false;
             }
         }
+
+        public void ShouldHaveValidationErrorFor<TDto>(TDto dto, string value)
+        {
+            throw new NotImplementedException();
+        }
     }
 }
```

Comment:
[suggestion] Implement this method and we should be able to uncomment the failing tests.